core/invoker/src/main/scala/whisk/core/invoker/InvokerReactive.scala

```diff
@@ -144,7 +154,7 @@ class InvokerReactive(
         // action will not hit in the cache due to change in the revision id;
         // if the doc revision is missing, then bypass cache
         if (actionid.rev == DocRevision.empty) {
-            logging.error(this, s"revision was not provided for ${actionid.id}")
+            logging.warn(this, s"revision was not provided for ${actionid.id}")
```

Comment:
See description. The InvokerHealth protocol deliberately sends messages without revision to force db access, which produces error logs.